src/main/java/Ui.java

```diff
@@ -0,0 +1,115 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+
+public class Ui {
+    private BufferedReader reader;
+
+    public Ui() {
+        this.reader = new BufferedReader(new InputStreamReader(System.in));
+    }
+
+    void intro() {
+        System.out.println("Hey there! This is Kums");
+        System.out.println("How can i help?");
+    }
+
+    void showLine() {
+        System.out.println("---------------------------------------");
+    }
+
+    String showError(String message) {
+        return message;
+    }
+
+    String bye() {
+        return "Bye. Hope to see you again soon!";
+    }
+
+    String readCommand() throws IOException {
+        return this.reader.readLine();
+    }
+
+    String showLoadingError() {
+        return "OOPS! The file hasn't been created yet";
+    }
+
+
+    /**
+     * Prints lines to show the user what task has been added.
+     *
+     * @param tasks TaskList of all the tasks
+     */
+    String addedTask(TaskList tasks) {
```

Comment:
Most of the functions in this class act as a printer. Hence, maybe these functions should be renamed as e.g. "printAddedTask"? Otherwise, it's not easy to know "addedTask" (and other functions in this class) are actually printers.